scripts/babylon-integration/utils/teardown.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -uo pipefail
+
+source "./common.sh"
+
+# Get the consumer FP address
+CONSUMER_FP_DIR=/home/finality-provider/.fpd
+KEYRING_FILENAME=$(ls $CONSUMER_FP_DIR/keyring-test | grep '\.address$' | sed 's/\.address$//')
+echo "Keyring filename: $KEYRING_FILENAME"
+CONSUMER_FP_ADDRESS=$(babylond keys parse "$KEYRING_FILENAME" --output json | jq -r '.formats[0]')
+echo "Consumer FP address: $CONSUMER_FP_ADDRESS"
+
+# Get the prefunded key address
+KEYRING_DIR=/home/.babylond
+PREFUNDED_ADDRESS=$(babylond keys show "$BABYLON_PREFUNDED_KEY" \
+    --keyring-dir "$KEYRING_DIR" \
+    --keyring-backend test \
+    --output json \
+    | jq -r '.address')
+echo "Prefunded address: $PREFUNDED_ADDRESS"
+
+# Check remaining balance
+CONSUMER_FP_BALANCE=$(babylond query bank balances "$CONSUMER_FP_ADDRESS" \
+    --node "$BABYLON_RPC_URL" \
+    -o json | jq -r '.balances[0].amount')
+echo "Consumer FP balance: $CONSUMER_FP_BALANCE"
+
+# If balance is less than gas transfer cost, don't send funds
+TRANSFER_GAS_COST=1000
+if [ "$CONSUMER_FP_BALANCE" -lt "$TRANSFER_GAS_COST" ]; then
+    echo "Consumer FP balance is less than gas transfer cost, skipping funds transfer"
+    exit 0
+fi
+
+# Otherwise, send out funds to prefunded key
+# Reserve 0.001 bbn = 1000 ubbn for gas
+CONSUMER_FP_KEYRING_DIR=$KEYRING_DIR/$CONSUMER_FINALITY_PROVIDER_KEY
+AMOUNT_TO_SEND=$((CONSUMER_FP_BALANCE - TRANSFER_GAS_COST))
+echo "Sending $AMOUNT_TO_SEND ubbn to prefunded key..."
+echo "Consumer FP keyring dir: $CONSUMER_FP_KEYRING_DIR"
+SEND_TX_HASH=$(babylond tx bank send \
+    ${CONSUMER_FINALITY_PROVIDER_KEY} \
+    ${PREFUNDED_ADDRESS} \
+    "${AMOUNT_TO_SEND}ubbn" \
+    --keyring-dir $CONSUMER_FP_KEYRING_DIR \
+    --keyring-backend test \
+    --chain-id $BABYLON_CHAIN_ID \
+    --node $BABYLON_RPC_URL \
+    --gas auto \
+    --gas-adjustment 1.5 \
+    --gas-prices 0.2ubbn \
+    --dry-run \
```

Comment:
we should remove it, right? 

if we want to check the output, we can update it:
```
SEND_TX_OUTPUT=$(babylond tx bank send \
    ${CONSUMER_FINALITY_PROVIDER_KEY} \
    ${PREFUNDED_ADDRESS} \
    "${AMOUNT_TO_SEND}ubbn" \
    --keyring-dir $CONSUMER_FP_KEYRING_DIR \
    --keyring-backend test \
    --chain-id $BABYLON_CHAIN_ID \
    --node $BABYLON_RPC_URL \
    --gas auto \
    --gas-adjustment 1.5 \
    --gas-prices 0.2ubbn \
    -o json -y)
echo "$SEND_TX_OUTPUT"
SEND_TX_HASH=$(echo "$SEND_TX_OUTPUT" | jq -r '.txhash')
```